tests/e2e/default/old-navigation/old-navigation.wdio-spec.js

```diff
@@ -43,7 +41,7 @@ describe('Old Navigation', () => {
     permissions.can_view_old_navigation = offlineUser.roles;
     await utils.updateSettings({ tasks, permissions }, true);
 
-    await loginPage.login(offlineUser);
+    await oldNavigationPage.login(offlineUser);
```

Comment:
This will skip the commonPage.waitForPageLoaded and then you can call the oldNavigationPage.waitForPageLoaded
```suggestion
    await loginPage.login({ ...offlineUser, loadPage: false });
    await oldNavigationPage.waitForPageLoaded();
```

Reply:
Like this is working:
<img width="700" alt="Screenshot 2024-09-13 at 3 18 22 PM" src="https://github.com/user-attachments/assets/4aa45fc7-5568-48f7-acd0-f7a42a92e2ee">